### DISCUSSION TITLE
Restyle own programs so they match the builtin programs

### DISCUSSION THREAD

**Discussion by lsolesen** [astashov/liftosaur#200] [Ideas]
When I am on https://www.liftosaur.com/user/programs my custom programs are styled like this. This looks fine. However, it is not possible to see how much time the single days will take, like it seems is calculated on the built in programs?

![image](https://github.com/user-attachments/assets/2f38738f-0466-4058-b0ba-cc0f01f2bf03)

![image](https://github.com/user-attachments/assets/5f5a5c02-8819-4f90-aeee-c82c8c7fa44f)

When in the app on https://www.liftosaur.com/app/ looking at programs, there is very little information about Your programs.

![image](https://github.com/user-attachments/assets/b2c508e7-f588-41e2-8b66-d5f789585d75)

Maybe the exercises, descripts, weeks, days, and so on can be added here as well?

**Comment by astashov** [comment]
Done
